pip/pip-275.md

```diff
@@ -0,0 +1,59 @@
+<!--
```

Comment:
If you're done can you please remove all comments?

Reply:
OK